docs/running-workflows/restarting-workflows/index.md

```diff
@@ -0,0 +1,116 @@
+# Restarting workflows
+
+In the [executing workflows](../executing-workflows/) page we discussed how to run and execute workflows.
+REANA gives users the possibility to *restart* a workflow as well.
+To restart a workflow, use the  `reana-client restart` command:
+
+```console
+$ reana-client restart -w my-physics-analysis.2
+==> SUCCESS: my-physics-analysis.2.1 has been queued
+```
+
+This presents some important differences compared to re-running a workflow one more time.
+
+## Differences between running and restarting a workflow
+
+### Run number
```

Comment:
I can see the description of the run number of a workflow either here in this page, or somewhere else in the documentation, such as in the "what is a workflow?" or "defining workflows" pages. Any opinion on this?